client.go

```diff
@@ -1135,6 +1135,13 @@ func min(a, b int) int {
 	return a
 }
 
+func notExistOrErr(err error) error {
```

Comment:
@mdlayher I have a feeling that there will be more error codes to translate, so one function for all of them sounds like a good plan.